manage_filter_edit_update.php

```diff
@@ -0,0 +1,107 @@
+<?php
+# MantisBT - A PHP based bugtracking system
+
+# MantisBT is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# MantisBT is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with MantisBT.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Manage filter edit update page
+ *
+ * @package MantisBT
+ * @copyright Copyright 2016  MantisBT Team - mantisbt-dev@lists.sourceforge.net
+ * @link http://www.mantisbt.org
+ *
+ * @uses core.php
+ * @uses filter_api.php
+ */
+
+require_once( 'core.php' );
+require_api( 'filter_api.php' );
+
+auth_ensure_user_authenticated();
```

Comment:
Is there a form token for this action page?  If not, should it be added? If yes, where is the validation?